tests/src/system/basic/WskBasicTests.scala

```diff
@@ -334,6 +334,16 @@ class WskBasicTests
             }
     }
 
+    it should "display a trigger summary when --summary flag is used with 'wsk trigger get'" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val triggerName = "mySummaryTrigger"
+            assetHelper.withCleaner(wsk.trigger, triggerName, confirmDelete = false) {
+                (trigger, name) =>
+                    trigger.create(name)
+            }
+            val stdout = wsk.trigger.get(triggerName, summary = true).stdout
+            stdout should include regex (s"(?i)trigger\\s+.*/${triggerName}")
```

Comment:
Can we make this a bit more narrow? We do know the namespace (do we? it might also be `_`) so we can assert the namespace. Case-insensitivity is debatable i guess.

i.e. `s"trigger\\s+/${wskprops.namespace}/${triggerName}"`